apps/api/src/app/subscribers/usecases/update-subscriber/update-subscriber.usecase.ts

```diff
@@ -38,6 +38,22 @@ export class UpdateSubscriber {
       updatePayload.avatar = command.avatar;
     }
 
+    if (command.channel != null) {
+      if (command.channel.integrationId != null) {
+        updatePayload['channel.integrationId'] = command.channel.integrationId;
+      }
+
+      if (command.channel.credentials != null) {
+        if (command.channel.credentials.channelId != null) {
+          updatePayload['channel.credentials.channelId'] = command.channel.credentials.channelId;
+        }
+
+        if (command.channel.credentials.accessToken != null) {
+          updatePayload['channel.credentials.accessToken'] = command.channel.credentials.accessToken;
+        }
+      }
+    }
```

Comment:
This one should allow multiple providers on the same subscriber object. uniquely by the integration identifier, so for now only a single slack should exist

Reply:
Resolved in 0bcabdd5b03c5d7edac3671ab2d1bd1ffa7ca814